### DISCUSSION TITLE
Is the response curve constructed using ad-stocked media or with original spend media?

### DISCUSSION THREAD

**Discussion by AruunnAnanda** [google/lightweight_mmm#145] [Q&A]
Hi there,

I would like to know if the X-axis of response curve depicts ad-stocked media or original spend media? 

![image](https://user-images.githubusercontent.com/107944288/212917004-38f604b9-27cd-44ac-bb5c-f203a86dea74.png)

Thanks!

**Comment by michevan** [comment]
Thanks a lot for flagging this! I just wanted to share a link to our new "[support](https://github.com/google/lightweight_mmm#support)" section in the readme to give you some more information on how we can and cannot support such questions at the moment.

**Comment by steven-struglia** [comment]
I believe it's using the original spend data looking at `lightweight_mmm.py` `self.media = media` and `plot.py` `media = media_mix_model.media` in the `plot_response_curves` function.

**Reply by slavakx** [reply]
I didn't debug the code, so I don't know if it includes the original or adstock spend but I think it should include the adstocked spend because we should see the effect of adstock on our spend

**Reply by steven-struglia** [reply]
I'm unsure if the model ever changes the `media_mix_model.media` parameter through the transformation functions besides the scaler used prior to feeding the data into the model. Though I do agree that we should be able to see the effect of adstock on spend inside the saturation curve, we should be able to see both ideally. Some sort of a parameter inside of the saturation curve function would be great to have. I have an edited saturation curve function that has a confidence interval on the spend vs. KPI curves that I'll share soon, and I can add an argument to the function like `transform_adstock = True or False` so we can do both.
